### DISCUSSION TITLE
Unable to deploy next js project [SyntaxError: Cannot use import statement outside a module]

### DISCUSSION THREAD

**Discussion by manassahoo-dev** [vercel/vercel#5808] [Help]
22:28:05.805  	> Build error occurred
22:28:05.806  	/vercel/workpath0/node_modules/@ctrl/tinycolor/dist/module/conversion.js:1
22:28:05.806  	import { bound01, pad2 } from './util';
22:28:05.806  	^^^^^^
**22:28:05.806  	SyntaxError: Cannot use import statement outside a module**
22:28:05.806  	    at wrapSafe (internal/modules/cjs/loader.js:915:16)
22:28:05.806  	    at Module._compile (internal/modules/cjs/loader.js:963:27)
22:28:05.806  	    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
22:28:05.806  	    at Module.load (internal/modules/cjs/loader.js:863:32)
22:28:05.806  	    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
22:28:05.807  	    at Module.require (internal/modules/cjs/loader.js:887:19)
22:28:05.807  	    at require (internal/modules/cjs/helpers.js:74:18)
22:28:05.807  	    at Object.<anonymous> (/vercel/workpath0/node_modules/@ant-design/colors/dist/index.js:5:18)
22:28:05.807  	    at Module._compile (internal/modules/cjs/loader.js:999:30)
22:28:05.807  	    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
22:28:05.807  	    at Module.load (internal/modules/cjs/loader.js:863:32)
22:28:05.807  	    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
22:28:05.807  	    at Module.require (internal/modules/cjs/loader.js:887:19)
22:28:05.807  	    at require (internal/modules/cjs/helpers.js:74:18)
22:28:05.807  	    at Object.<anonymous> (/vercel/workpath0/node_modules/@ant-design/icons/lib/utils.js:22:15)
22:28:05.807  	    at Module._compile (internal/modules/cjs/loader.js:999:30)
22:28:05.852  	error Command failed with exit code 1.
22:28:05.852  	info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
22:28:05.863  	Error: Command "yarn run build" exited with 1
22:28:10.457  	Done with "package.json"

**Comment by manassahoo-dev** [comment]
Link
[https://vercel.com/cforcode/nextjs/2uqrhnxhk](url)

**Comment by fsultani-songclip** [comment]
Same error here.

**Comment by ardifirmansyah8** [comment]
yeah, same error

**Comment by ardifirmansyah8** [comment]
i solved it referring to this thread https://github.com/scttcper/tinycolor/issues/208

adding this code to package.json:

```
"resolutions": {
    "@ant-design/colors": "5.0.0"
  }
```

**Reply by manassahoo-dev** [reply]
Thanks a lot @ardifirmansyah8. Able to deploy now. Thank you so much

**Comment by fsultani-songclip** [comment]
Now it works for me, too.  I didn't change anything in package.json, so I'm guessing it was an internal fix.  I usually delete `node_modules` and `package-lock.json`, which wasn't fixing it earlier, so maybe an internal fix was recently deployed.

**Reply by naganathH** [reply]
I think it was fixed internally, I faced same issue yesterday.
